app/models/manageiq/providers/vmware/infra_manager/vapp.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::Vmware::InfraManager::Vapp < Vapp
```

Comment:
Yes modeling as orchestration stacks are the other option we want to play around with
Agree with `VirtualApp` over `Vapp`